src/components/CocktailList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Cocktail from './Cocktail';
+import Loading from './Loading';
+import Pagination from './Pagination';
+import { useGlobalContext } from '../context';
+
+const CocktailList = () => {
+  const { currentCocktail, cocktailsPerPage, cocktails, loading, paginate } = useGlobalContext();
+  if(loading) {
+    return <Loading />
+  }
+  if(currentCocktail.length < 1) {
```

Comment:
a shorter way is `if(!currentCocktail.length)`. It's not super important though but it just makes things a little bit briefer